andela/getmyshop
# 107709190 Feature: Shop creation form page

#### What does this PR do?

Shop owners should be able to see the shop creation page with friendly UI
#### Description of Task to be completed?

As a logged in user
I should see a nice user interface of the shop creation page
And a form for the shop creation
So that the shop creation process would be easy.
#### How should this be manually tested?

When user clicks on the create shop link in the top bar
Then the user should see the form for creating a new shop
#### Any background context you want to provide?

Given that, I'm a logged in user
When I visit the shop creation page
Then I should see a nice user interface.
#### What are the relevant pivotal tracker stories?

```
#117064357
```
